datadog_checks_dev/datadog_checks/dev/tooling/templates/docs/default.yaml

```diff
@@ -0,0 +1,199 @@
+# TODO: To be split into separate files
+
+- template: overview
```

Comment:
Since these are templates, I think there's no need to have a template name but rather we should go by fs paths

Reply:
Agreed - the `template` name here should be read as file paths. I'll change them to comments.